### DISCUSSION TITLE
standalone service

### DISCUSSION THREAD

**Discussion by demosspro** [supersaiyanmode/PyWebOSTV#98] [Q&A]
Hello! Thank you for work, can you help me understand, how i can use it in standalone mode? i wanna start it in a small linux board that accept command from tcp at simple text command and control tv. It is need async functions, i think?

**Comment by supersaiyanmode** [comment]
Should be straightforward I think. Can you please share what you've tried so far?

**Reply by demosspro** [reply]
Yes, sure. Please look at the file MY.PY in my repository https://github.com/demosspro/PyWebOSTV
Thanks.
